Packages/vcs/Lib/vcs2vtk.py

```diff
@@ -94,11 +94,27 @@ def putMaskOnVTKGrid(data, grid, actorColor=None, cellData=True, deep=True):
             grid.GetPointData().AddArray(ghost)
     return mapper
 
+def handleProjectionEdgeCases(projName, data):
+    # For mercator projection, latitude values of -90 or 90
+    # transformation result in infinity values. We chose -85, 85
```

Comment:
@jbeezley sure, but matplotlib uses 85.0 and so as some other tools. For UV-CDAT since we scale the plot, which makes it a rectangle anyways but I am willing to change it to 85.051129 if that is preferred.